specs/latest/2.0/index.html

```diff
@@ -1092,6 +1094,52 @@ <h2 class="no-toc">Table of contents</h2>
         it is the responsibility of the WebGL API to ensure that data are accessed consistently. This applies
         even if the buffer is currently bound to a transform feedback binding point.
       </dd>
+      <dt>
+        <p class="idl-code">Promise&lt;ArrayBuffer&gt; getBufferSubDataAsync(GLenum target, GLintptr offset, ArrayBuffer returnedData)
+        </p>
+      </dt>
+      <dd>
+        For the WebGLBuffer bound to the passed target, read <code>returnedData.byteLength</code>
+        bytes from the buffer starting at byte offset <code>offset</code> and write them
+        to <code>returnedData</code>. This command executes serially with respect to others in the
+        command stream, but the data is only written to <code>returnedData</code> when the promise
+        is fulfilled.
+
+        <br><br>
+        If any of the following error conditions occur:
+
+        <ul>
+          <li><code>offset + returnedData.byteLength</code> would extend beyond the end of the buffer</li>
```

Comment:
"the buffer" is not clear.   Perhaps you should formally define an identifier somewhere above.  Something like ` Let <var>buffer</var> be the WebGLBuffer bound to <code>target</code>.`